modules/admin/proc-customizing-telemetry-segment-using-helm.adoc

```diff
@@ -0,0 +1,42 @@
+[id="proc-customizing-telemetry-segment-using-helm_{context}"]
+= Customizing telemetry Segment source using the Helm Chart
+
+You can configure integration with your Segment source by using the Helm Chart.
+
+.Prerequisites
+
+* You have logged in as an administrator in the {ocp-short} web console.
+* You have installed {product} on {ocp-short} using the Helm Chart.
```

Comment:
@shipsing Just a thought - Can we add the cross-reference link of installation procedure here?